Add additional steps to erc 20 process. (#2232)
filecoin-project/filecoin-docs: smart-contracts/fundamentals/erc-20-quickstart.md

In Remix, workspaces are where you can create a contract, or group of contracts, for each project. Let’s create a new workspace to create our new ERC-20 token.

1. Open [remix.ethereum.org](https://remix.ethereum.org).

> 2. Open the drop down menu and click `create new workspace`.

Reviewer: Should it be `dropdown` or `drop-down`?

> 2. Open the drop down menu and click `create new workspace`.

Reviewer: ```suggestion
2. Open the dropdown menu and click `create new workspace`.
```

    ![Create a new workspace.](../../.gitbook/assets/smart-contracts-fundamentals-erc-20-quickstart-remix-dropdown.webp)

> 3. In the **Choose a template** dropdown, select **ERC 20**.

Reviewer: ```suggestion
3. In the **Choose a template** dropdown, select **ERC20**.
```

> 4. Select the **Mintable** checkbox.

Reviewer: ```suggestion
4. Under **Customize template** > **Features**, check the **Mintable** box.
```
